src/currency/currency-formatter.ts

```diff
@@ -57,22 +57,16 @@ export default class CurrencyFormatter {
 
     private _formatNumber(amount: number): string {
         const positiveAmount = Math.abs(amount);
-        const [ integerAmount, decimalAmount = '' ] = positiveAmount.toString().split('.');
+        const [ integerAmount, decimalAmount = '' ] = (positiveAmount.toFixed(this._decimalPlaces)).toString().split('.');
```

Comment:
I remember `toFixed` is actually not a reliable way to round decimals. For example `859.385.toFixed(2)` returns `859.38` instead of `859.39`. I found this [StackOverflow post](https://stackoverflow.com/questions/10015027/javascript-tofixed-not-rounding) which might help you to overcome this issue.